proof/invariant-abstract/Ipc_AI.thy

```diff
@@ -2101,6 +2101,7 @@ crunches maybe_donate_sc
   and caps_of_state[wp]: "\<lambda>s. P (caps_of_state s)"
   and interrupt_irq_node[wp]: "\<lambda>s. P (interrupt_irq_node s)"
   and typ_at[wp]: "\<lambda>s. P (typ_at T t s)"
+  and tcb_at[wp]: "tcb_at t"
```

Comment:
You can get this from an `abs_typ_at_lifts` (though maybe that's not quite the right name)

Reply:
Or maybe it's more convenient so that a lemma is made for the subfunctions, in which case I don't mind this